front/components/vaults/VaultSideBarMenu.tsx

```diff
@@ -110,9 +110,102 @@ const SubItemIconItemWrapper = (
 
 // System vault.
 
-const SystemVaultMenu = () => {
-  // TODO(Groups UI) Implement system vault menu.
-  return <></>;
+const SYSTEM_VAULTS_ITEMS = [
```

Comment:
Should we move this to types? Feel like we are gonna need this in other places

Reply:
This is specifically for menu rendering so I'd keep it close to the implementation. What other places do you have in mind?

Reply:
The company data screen?